dapps/assets/src/i18n/en.json

```
@@ -43,7 +43,8 @@
       "subject-placeholder": "Subject",
       "subject-prefill": "Contact Request",
       "updated": "Updated",
-      "users": "Users"
+      "users": "Users",
+      "automatic-share": "Choose data sections of your profile that should be automatically shared with the new contact."
```

I think like this it may cause less confusion:

```suggestion
      "automatic-share": "Choose the sections of your profile that should be shared with the new contact."
```